src/main/java/dev/jbang/Cache.java

```diff
@@ -11,7 +11,7 @@
 public class Cache {
 
 	public enum CacheClass {
-		urls, jars, jdks, projects, scripts, stdins, deps
```

Comment:
are `kotlins` any different than `jars` ?

Reply:
they get expanded out like jdks do for `kotlinc` access, etc.

Reply:
Ah. Sure.

Reply:
Is it called kotlin development kits ? So kdks?

Reply:
`kotlincs' ? as its the kotlin compiler or is it more ?

Reply:
pushed update to use `kotlincs` and added the support in `cache clear` command.